configfs.c

```diff
@@ -186,6 +186,17 @@ char *tcmu_get_cfgfs_str(const char *path)
 	if (ret == 0)
 		return NULL;
 
+	/*
+	 * Some files like members will with a null char at the end of each
+	 * member, just replace it with '\n', except the last one.
```

Comment:
Just missing a word. I think we want:

Some files like members will terminate each member/line with a null char. Except for the last one, replace it with '\n' so parsers will just see an empty member.

Reply:
Yes, will replace it.